app/src/main/java/io/intrepid/contest/screens/contestcreation/categorieslist/CategoriesListPresenter.java

```diff
@@ -19,6 +19,7 @@
 
     @Override
     public void displayCategories(Contest contest) {
+        if(view == null) return;
```

Comment:
If there a larger problem we need to look at? What is allowing the view to be negative here? We may need to reorder some of the function calls.